test/utils/TransactionSender.py

```diff
@@ -17,17 +17,8 @@ async def send_transaction(self, calls, signers, nonce=None, max_fee=0):
             execution_info = await self.account.get_nonce().call()
             nonce = execution_info.result.nonce
 
-        call_inputs = []
-        calls_with_selector = []
-        calldata = []
-        for i in range(len(calls)):
-            if len(calls[i]) != 3:
-                raise Exception("Invalid call parameters")
-            call = calls[i]
-            call_input = (call[0], get_selector_from_name(call[1]), len(calldata), len(call[2]))
-            call_inputs.append(call_input)
-            calldata.extend(call[2])
-            calls_with_selector.append((call[0], get_selector_from_name(call[1]), call[2]))
+        calls_with_selector = map(lambda call: (call[0], get_selector_from_name(call[1]), call[2]), calls)
```

Comment:
`map(lambda: ...)` is an antipattern in Python. Always prefer list comprehensions:
```suggestion
        calls_with_selector = [(call[0], get_selector_from_name(call[1]), call[2]) for call in calls]
```